### PR TITLE
[mysql] use `information_schema` when < 5.6.0

### PR DESCRIPTION
**Pull request by yannmh**
The `performance_schema.threads` table is not available on MySQL <
`5.6.0`. As a result, retrieving the slaves' statuses triggers the
following error:

```
ProgrammingError: (1146, u"Table 'performance_schema.threads' doesn't
exist")
```

This is a regression introduced by
https://github.com/DataDog/dd-agent/pull/2603

### CODE REVIEW

File: checks.d/mysql.py [hunk 1]
```diff
@@ -863,12 +863,17 @@ def _get_replica_stats(self, db):
             self.warning("Privileges error getting replication status (must grant REPLICATION CLIENT): %s" % str(e))
             return {}
 
-    def _get_slave_status(self, db, nonblocking=True):
+    def _get_slave_status(self, db, above_560):
```
Comment (truthbk) [inline checks.d/mysql.py]: should we keep the non-blocking optional parameter?

Comment (yannmh) [inline checks.d/mysql.py]: Not sure what's the benefits in doing that. `_get_slave_status` is only called here and once.

Comment (truthbk) [inline checks.d/mysql.py]: I'm not sure if that was added on a customer request, not specifically, but asking to make the check more performant. But you are correct, it's only holding the mutex once per check run.

Comment (truthbk) [inline checks.d/mysql.py]: Your call.

Comment (truthbk) [inline checks.d/mysql.py]: 👍

**Pull request merged**